process/champion.md

```diff
@@ -0,0 +1,37 @@
+# Responsibilities of a Champion
+
+Champions are authors and editors of proposals. The champion is responsible for
+the evolution of the proposal from Phase 0 through Phase 5, at which point
+maintenance transfers to the WebAssembly Working Group (WG). Champions have
+admin permissions in the proposal repository. Periodically, champions may bring
+their proposal to the [Community Group (CG)
+meetings](https://github.com/WebAssembly/meetings/) to ask for consensus on
+stage advancement.
+
+When asking for advancement, the champion is expected to make the whole
+proposal accessible for review by the CG, by explaining its contents, providing
+supporting documentation, etc. Material changes should be presented explicitly.
+
+It is often beneficial for champions to keep the CG updated with periodic
+status updates outlining major changes in the regular [CG
+meetings](https://github.com/WebAssembly/meetings/). While minor changes can be
+made through discussions within the repository or the relevant subgroup,
+substantial design changes may require a vote from the CG so that the CG has a
+chance to review the proposal and possibly re-evaluate if the proposal is in
+the appropriate stage. Examples include a change to the fundamental assumption
+or structure of a proposal, a decision that has a substantial impact on the
+future direction, or a resolution to a contentious issue that is hard to be
+resolved.
+
+It is not necessary that champions are knowledgeable about every component of
+the spec or its implementation, but they are responsible for the general
+management of the proposal repository, which includes:
```

Comment:
This section should probably also mention shared responsibilities with subgroup chairs. As this varies between groups, we could keep it fairly high level and add a note that these can be shared responsibilities with the subgroup chair when the champion and the subgroup chair are not the same person.

cc @tlively and @linclark for visibility.

Reply:
+1, a sentence stating that these responsibilities may be shared with subgroup chairs in cases where there is a relevant subgroup sounds good to me.

Reply:
Done